CHANGELOG.md

```diff
@@ -59,6 +105,12 @@ releases.
 
 ### Fixed
 
+- **CUMULUS-2854**
+  - Fixed queue granules behavior where the task was not accounting for granules that
+  *already* had createdAt set. Workflows downstream in this scenario should no longer
+  fail to write their granules due to order-of-db-writes constraints in the database
+  update logic.
+
```

Comment:
This should be under the release notes for 10.1.1

Reply:
Updated as comment. Thanks